src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -35,12 +35,25 @@
 import java.util.Map;
 import org.apache.kafka.common.config.ConfigException;
 import org.apache.kafka.common.config.types.Password;
+import org.elasticsearch.common.unit.ByteSizeValue;
 
 import static org.apache.kafka.common.config.ConfigDef.Range.between;
 import static org.apache.kafka.common.config.SslConfigs.SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_CONFIG;
 import static org.apache.kafka.common.config.SslConfigs.addClientSslSupport;
 
 public class ElasticsearchSinkConnectorConfig extends AbstractConfig {
+  private static final long MEGABYTES_TO_BYTES = 1024 * 1024;
+
+  public static final String MAX_BATCH_SIZE_BYTES_CONFIG = "max.batch.size.bytes";
+  private static final String MAX_BATCH_SIZE_BYTES_DOC =
+          "The maximum size (in bytes) of the bulk request to be processed by the Connector when"
+          + " writing records to Elasticsearch. The default is 10mb, the maximum setting possible."
+          + "If set to -1 or the condition for batch size by number of records is met first, this "
+          + "configuration will be ignored and the setting for batch size by number of records "
+          + "would be used instead.";
+  private static final long MAX_BATCH_SIZE_BYTES_DEFAULT = 10 * MEGABYTES_TO_BYTES;
+  private static final long MIN_BATCH_SIZE_BYTES_DEFAULT = -1;
+  private static final String BATCH_SIZE_BYTES_DISPLAY = "Max Batch Size in Bytes";
```

Comment:
this should go under `BATCH_SIZE_CONFIG`

Reply:
and for consistency, let's rename these prefixes to `BULK_SIZE_`

Reply:
I indicated in the var name it's in bytes, is that necessary? Or should it just be `BULK_SIZE`?